GenerateEverything.hs

```diff
@@ -42,10 +42,11 @@ unsafeModules = map modToFile
   , "Codata.Musical.Covec"
   , "Codata.Musical.Conversion"
   , "Codata.Musical.Stream"
-  , "Data.PostulatedReal"
-  , "Data.PostulatedReal.Base"
-  , "Data.PostulatedReal.Propertis"
-  , "Data.PostulatedReal.Propertis.Core"
+  , "Data.ParametrizedReal"
+  , "Data.ParametrizedReal.Base"
+  , "Data.ParametrizedReal.Interface"
+  , "Data.ParametrizedReal.Propertis"
+  , "Data.ParametrizedReal.Propertis.Core"
```

Comment:
```suggestion
  , "Data.ParametrizedReal.Properties"
  , "Data.ParametrizedReal.Properties.Core"
```

Reply:
Although I guess it should not be in the list of unsafe modules anymore if it's `--safe`

Reply:
Yes, you are absolutely right.